lib/oli/analytics/datashop.ex

```diff
@@ -78,6 +78,7 @@ defmodule Oli.Analytics.Datashop do
         date: hd(part_attempts).activity_attempt.resource_attempt.inserted_at,
         email: email,
         sub: sub,
+        datashop_session_id: hd(part_attempts).datashop_session_id,
```

Comment:
For existing part attempt records, this `datashop_session_id` will be `null`.  It seems like we should handle this case here and generate a UUID, otherwise the `<meta>` element will have a null session id.

Reply:
fixed